files/ko/web/html/index.html

```diff
@@ -1,117 +1,197 @@
+  {{HTMLElement("header")}}, {{HTMLElement("footer")}}, {{HTMLElement("article")}}, {{HTMLElement("section")}},
+  {{HTMLElement("p")}}, {{HTMLElement("div")}}, {{HTMLElement("span")}}, {{HTMLElement("img")}},
+  {{HTMLElement("aside")}}, {{HTMLElement("audio")}}, {{HTMLElement("canvas")}}, {{HTMLElement("datalist")}},
+  {{HTMLElement("details")}}, {{HTMLElement("embed")}}, {{HTMLElement("nav")}}, {{HTMLElement("output")}},
+  {{HTMLElement("progress")}}, {{HTMLElement("video")}}, {{HTMLElement("ul")}}, {{HTMLElement("ol")}},
+  {{HTMLElement("li")}} 등 많은 종류가 존재합니다.
+</p>
+
+<p>
+  HTML 요소는 "태그"를 사용해서 문서의 다른 텍스트와 구분합니다. 태그는 "<code>&lt;</code>", 태그 이름,
+  "<code>&gt;</code>"로 이루어집니다. 태그 안의 요소 이름은 대소문자를 구분하지 않습니다. 즉, 대문자, 소문자, 아니면
+  섞어서도 작성할 수 있습니다. 예를 들어 <code>&lt;title&gt;</code>요소는 <code>&lt;Title&gt;</code>,
+  <code>&lt;TITLE&gt;</code>, 그 외 기타 가능한 모든 방법으로 작성할 수 있습니다.
+</p>
+
+<p>아래 문서에서 HTML을 더 깊게 배워보세요.</p>
+
+<h2 id="key_resources">중요한 자료</h2>
 
-<p>HTML 요소는 문서의 다른 텍스트와 "태그"로 구분합니다. 태그는 "<code>&lt;</code>", 태그 이름, "<code>&gt;</code>"로 이루어집니다. 태그 안의 요소명은 대소문자를 구분하지 않습니다. 즉, 대문자, 소문자, 아니면 섞어서도 작성할 수 있습니다. 예를 들어 <code>&lt;title&gt;</code>요소는 <code>&lt;Title&gt;</code>, <code>&lt;TITLE&gt;</code>, 그 외 기타 가능한 모든 방법으로 작성할 수 있습니다.</p>
+<dl>
+  <dt>HTML 소개</dt>
+  <dd>
+    웹 개발이 처음이시라면 <a href="/ko/docs/Learn/Getting_started_with_the_web/HTML_basics">HTML 기본</a> 문서에서
+    HTML이란 무엇인지, 그리고 HTML의 사용법을 배워보세요.
+  </dd>
+  <dt>HTML 자습서</dt>
+  <dd>
+    <a href="/ko/docs/Learn/HTML">HTML 학습지</a>를 방문하시면 HTML 사용법과 자습서, 완전한 예제를 찾아보실 수 있습니다.
+  </dd>
+  <dt>HTML 참고서</dt>
+  <dd>
+    방대한 <a href="/ko/docs/Web/HTML/Reference">HTML 참고서</a>에서는 HTML을 구성하는 모든 요소와 특성의 자세한 설명을
+    볼 수 있습니다.
+  </dd>
+</dl>
 
-<p>아래에서 HTML을 더 깊이 배워보세요.</p>
+<div class="callout">
+  <h4 id="looking_to_become_a_front-end_web_developer">프런트엔드 웹 개발자가 되고자 하시나요?</h4>
 
-<section class="cleared" id="sect1">
-<ul class="card-grid">
- <li><span>HTML 소개</span>
+  <p>목표를 달성하기 위한 모든 필수 정보가 들어있는 교육 과정을 준비했습니다.</p>
 
-  <p>웹 개발을 새로 시작하는 분이라면 <a href="/ko/docs/Learn/Getting_started_with_the_web/HTML_basics">HTML 기초</a>를 읽고, HTML이 무엇인지와 어떻게 사용하는지 배워보세요.</p>
- </li>
- <li><span>HTML 자습서</span>
-  <p>HTML 사용법, 자습서, 완전한 예제 등은 <a href="/ko/docs/Learn/HTML">HTML 익힘장</a>에서 확인할 수 있습니다.</p>
- </li>
- <li><span>HTML 참고서</span>
-  <p><a href="/ko/docs/Web/HTML/Reference">방대한 참고서</a>는 HTML을 구성하는 요소와 속성에 대한 자세한 내용을 담고 있습니다.</p>
- </li>
-</ul>
+  <p>
+    <a href="/ko/docs/Learn/Front-end_web_developer"><strong>시작하기</strong></a>
+  </p>
+</div>
 
-<div class="row topicpage-table">
-<div class="section">
-<h2 class="Tools" id="초보자_자습서">초보자 자습서</h2>
+<h2 class="Tools" id="beginners_tutorials">초보자 자습서</h2>
 
-<p>MDN의 <a href="/ko/docs/Learn/HTML">HTML 익힘장</a>은 HTML을 가장 기초부터 알려주는 여러 개의 장을 가지고 있습니다. 이전 경험은 필요하지 않습니다.</p>
+<p>
+  MDN <a href="/ko/docs/Learn/HTML">HTML 학습지</a>는 HTML을 처음부터 알려주는 모듈로 구성되어 있습니다. 사전 지식도
```

Comment:
```suggestion
  MDN <a href="/ko/docs/Learn/HTML">HTML 학습지</a>는 HTML을 처음부터 알려주는 모듈로 구성되어 있습니다. 사전 지식은
```

조금 더 자연스러워 보여서 리뷰했습니다.

Reply:
아, 이거 #1002 부터 "사전 지식은"으로 했다가 앞 문장이랑 잘 이어지지 않아서 "사전 지식도"로 맞춘건데... "사전 지식은"이 더 자연스러울까요?

Reply:
계속 읽어보니 "사전 지식도"가 자연스러워 보여요! 괜찮은것 같아요 :)